spec/components/navbar/navbar_component_spec.rb

```diff
@@ -0,0 +1,38 @@
+require "rails_helper"
+
+describe Navbar::NavbarComponent, type: "component" do
+  let(:sitemap) do {
+    markdown_content:
+    [
```

Comment:
This will require #598 to be merged (once the fixes to the release have gone out)

But after that - This can be 

```ruby
let(:all_frontmatter) do
  {
    "/accessibility", { title: "Accessibility", navigation: 20 },
    "/accessibility2", { title: "Second accessibility page", navigation: 30 },
  }
end

before do
  allow(Pages::Frontmatter).to receive(:list).and_return all_frontmatter
end
```

One thing to note is the index of markdown pages can only have one hash of frontmatter per path, because in reality there can only be one markdown file per path